libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -132,6 +133,12 @@ AP_BattMonitor::init()
                                                                  hal.i2c_mgr->get_device(_params[instance]._i2c_bus, AP_BATTMONITOR_SMBUS_I2C_ADDR,
                                                                                          100000, true, 20), 6);
                 break;
+            case AP_BattMonitor_Params::BattMonitor_TYPE_NeoDesign:
```

Comment:
Sorry to be nitpicky but could we make the order within this case statement to the enum order? .. so down at the bottom..

Reply:
Sure, no problem, it's moved.